hypervisor/src/mshv/mod.rs

```diff
@@ -597,6 +597,17 @@ impl cpu::Vcpu for MshvVcpu {
 
                     Ok(cpu::VmExit::Ignore)
                 }
+                hv_message_type_HVMSG_UNACCEPTED_GPA => {
+                    let info = x.to_memory_info().unwrap();
+                    let gva = info.guest_virtual_address;
+                    let gpa = info.guest_physical_address;
+
+                    Err(cpu::HypervisorCpuError::RunVcpu(anyhow!(
+                        "Unhandled VCPU exit: Unaccepted GPA({:x}) found at GVA({:x})",
+                        gpa,
+                        gva,
+                    )))
```

Comment:
I am not sure if `hv_message_type_HVMSG_UNACCEPTED_GPA` is actually being handled in this case, which is simply returning an Error that will cause the corresponding vcpu thread to terminate. 

This is pretty much the same behavior as before (where it will panic the vcpu thread), besides more debugging information for the Error. If this is mainly to add additional debug information, I'd rephase the commit message to clarify that.

Reply:
It's a fatal message from the hypervisor prospective and it will kill the vcpu thread. So I can add this information in the commit message.

Reply:
Fixed the commit message.